src/pages/index.js

```diff
@@ -1,106 +1,204 @@
+const editProfilePopup = new PopupWithForm(profileEditModal, handleProfileFormSubmit);
+const previewImagePopup = new PopupWithImage(previewImageModal);
+const addNewCardPopup = new PopupWithForm(cardModal, handleSubmitCard);
+const deleteImagePopup = new PopupWithConfirm(deleteCardModal);
+const editAvatarPopup = new PopupWithForm(profileAvatarModal, handleProfileAvatarSubmit);
 
 /* -------------------------------------------------------------------------- */
-/*                                   Profile                                  */
+/*                               Form Validators                              */
 /* -------------------------------------------------------------------------- */
 
-const userInfo = new UserInfo({
-  nameSelector: '.profile__title', 
-  jobSelector: '.profile__description'
-});
-
-const editProfilePopup = new PopupWithForm(profileEditModal, ({ title, description }) => {
-  userInfo.setUserInfo({ title, job: description });
-  editProfilePopup.close();
-});
-
-profileEditBtn.addEventListener('click', openProfilePopup);
+editProfileFormValidator.enableValidation();
+addCardFormValidator.enableValidation();
+avatarFromValidator.enableValidation();
+deleteCardFormValidator.enableValidation();
 
-function openProfilePopup() {
-  const { name, job } = userInfo.getUserInfo();
-  profileTitleInput.value = name;
-  profileDescriptionInput.value = job;
-  editFormValidator.toggleButtonState();
-  editProfilePopup.open();
-}
+/* -------------------------------------------------------------------------- */
+/*                                 Api Promise                                */
+/* -------------------------------------------------------------------------- */
 
-editProfilePopup.setEventListeners();
+let userId;
+let cardList;
+
+Promise.all([Api.getInitialCards(), Api.getUserInfo()])
+.then(([initialCards, userData]) => {
+  userId = userData._id;
+  userDataInfo.setUserInfo(userData.name, userData.about);
+  userInfo.setProfileAvatar(userData.avatar);
+  cardList = new Section(
+    {
+      items: initialCards,
+      renderer: (data) => {
+        const newCard = createCard(data);
+        cardList.addItem(newCard);
+      },
+    },
+    cardList
+  );
+  cardList.renderItems();
+})
+.catch(() => console.log(err));
 
 /* -------------------------------------------------------------------------- */
-/*                                Section Class                               */
+/*                              Profile Functions                             */
 /* -------------------------------------------------------------------------- */
 
-const cardListSection = new Section(
-  {
-    items: initialCards,
-    renderer: ({ name, link }) => {
-      const newCard = createCard({ name, link });
-      cardListSection.addItem(newCard);
-    },
-  },
-  cardListEl
-);
+function openProfilePopup() {
+  const { nameSelector, descriptionSelector} = userInfo.getUserInfo();
```

Comment:
Selectors are values, by which you search for the elements in the dom. Please rename the info values to `name` and `description` (without `selector` postfix)